docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1156 @@
+    :class:`~quantum_info.BaseOperator`.
+
+    2. :class:`~opflow.OperatorBase` also implements methods such as ``.to_matrix()`` or ``.to_spmatrix()``, which are only found
+    in some of the :class:`~quantum_info.BaseOperator` subclasses.
+
+    See API reference for more information.
+
+
+Operator Globals
+----------------
+*Back to* `Index`_
+
+Opflow provided shortcuts to define common single qubit states, operators, and non-parametrized gates in the
+:mod:`~qiskit.opflow.operator_globals` module.
+
+These were mainly used for didactic purposes or quick prototyping, and can easily be replaced by their corresponding
+:mod:`~qiskit.quantum_info` class: :class:`~qiskit.quantum_info.Pauli`, :class:`~qiskit.quantum_info.Clifford` or
+:class:`~qiskit.quantum_info.Statevector`.
+
+
+1-Qubit Paulis
+~~~~~~~~~~~~~~
+*Back to* `Index`_
+
+The 1-qubit paulis were commonly used for quick testing of algorithms, as they could be combined to create more complex operators
+(for example, ``0.39 * (I ^ Z) + 0.5 * (X ^ X)``).
+These operations implicitly created operators of type  :class:`~qiskit.opflow.PauliSumOp`, and can be replaced by
+directly creating a corresponding :class:`~qiskit.quantum_info.SparsePauliOp`, as shown in the example :ref:`below <1_q_pauli>`.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+   * - :class:`~qiskit.opflow.X`, :class:`~qiskit.opflow.Y`, :class:`~qiskit.opflow.Z`, :class:`~qiskit.opflow.I`
+     - :class:`~qiskit.quantum_info.Pauli`
+
+       ..  tip::
+
+           For direct compatibility with classes in :mod:`~qiskit.algorithms`, wrap in :class:`~qiskit.quantum_info.SparsePauliOp`.
+
+
+..  _1_q_pauli:
+
+Example 1: Defining the XX operator
+###################################
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.opflow import X
+
+    operator = X ^ X
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Alternative</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.quantum_info import Pauli, SparsePauliOp
+
+    X = Pauli('X')
+    op = X ^ X
+
+    # equivalent to:
+    op = SparsePauliOp('XX')
+
+.. raw:: html
+
+   </details>
+
+Example 2: Defining a more complex operator
+###########################################
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.opflow import I, X, Z
+
+    op = 0.39 * (I ^ Z ^ I) + 0.5 * (I ^ X ^ X)
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Alternative</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    op = SparsePauliOp.from_list([("IZI", 0.39), ("IXX", 0.5)])
+
+    # or...
+    op = SparsePauliOp.from_sparse_list([("Z", [1], 0.39), ("XX", [0,1], 0.5)], num_qubits = 3)
+
```

Comment:
One can also directly build it using its constructor
```
op = SparsePauliOp(["IZI", "IXX"], coeffs=[0.39, 0.5])
```

Reply:
added